src/api/spec/models/package_spec.rb

```diff
@@ -731,5 +731,29 @@
       it { expect(package.ignored_requests).to be_nil }
     end
   end
+
+  describe '#belongs_to_product?' do
+    context 'a product package (_product)' do
+      subject { create(:package, name: '_product') }
```

Comment:
[Nitpick] I think the subject could be `create(:package, name: '_product').belongs_to_product?`, then the following examples would be `it { is_expected ... }`

Reply:
I actuallyfind it a bit clearer as it is right now